katib/installs/katib-external-db/kustomization.yaml

```diff
@@ -11,6 +11,12 @@ secretGenerator:
   - name: katib-mysql-secrets
     envs:
       - secrets.env
-commonLabels:
-  app.kubernetes.io/component: katib
-  app.kubernetes.io/name: katib-controller
+patches:
```

Comment:
Why do we have this patch?
I tested locally with `commonLabels` and it also adds label to the Katib mysql Secret.

Reply:
@andreyvelich I did this because if I put `commonLabels` in this one, it will override the `commonLables` in `../../components/katib-crds`, which would make them incorrect. (As `app.kubernetes.io/name: katib-controller` is the wrong label for the CRDs)

Reply:
Got it, thanks @thesuperzapper